### DISCUSSION TITLE
Using Wrapper input wont work for me

### DISCUSSION THREAD

**Discussion by HadiSDev** [react-hook-form/react-hook-form#2424] [Q&A] [answered]
Hi

I have recently switched to RHF, and I am trying to use wrapper input controls using chakra-ui as shown in this codesandbox:
https://codesandbox.io/s/chakra-ui-react-hook-form-08hxb?file=/src/HookForm.js:335-550

However the errors list does not get updated when trying to submit, but if I try to uncomment the commented input field name so I have the two fields (name and test1), the errors list gets updated, and the error message is show.

```
      {/* <FormControl isInvalid={errors.name}>
        <FormLabel htmlFor="name">First name</FormLabel>
        <Input
          name="name"
          placeholder="name"
          ref={register({ validate: validateName })}
        />
        <FormErrorMessage>
          {errors.name && errors.name.message}
        </FormErrorMessage>
      </FormControl> */}
```

**Comment by bluebill1049** [comment]
Don't know where you getting that CSB, but it's really outdated v3 in the codesandbox, we on v6 right now: https://codesandbox.io/s/chakra-ui-react-hook-form-5k5rm?file=/src/HookForm.js

**Reply by HadiSDev** [reply]
@bluebill1049 thanks for noticing the old version👍  but I actually found my mistake. I was using errors.name in the isinvalid property

I should use errors[name] .... 🙈🙈

**Comment by HadiSDev** [comment, marked as answer]
https://codesandbox.io/s/chakra-ui-react-hook-form-9kxjs?file=/src/HookForm.js
